packages/flutter_tools/templates/create/.analysis_options.tmpl

```diff
@@ -0,0 +1,70 @@
+# This file contains the analysis options used by the Dart analyzer
+# to decide what errors, warnings, and hints should be shown.
+# See the configuration guide for more
+# https://github.com/dart-lang/sdk/tree/master/pkg/analyzer#configuring-the-analyzer
+
+analyzer:
+  language:
+    enableStrictCallChecks: true
+    enableSuperMixins: true
+  strong-mode: false
+  errors:
+    # we allow overriding fields (if they use super, ideally...)
```

Comment:
Why include these "strong_mode" options if strong mode is turned off in this options file?  It seems likely that these settings will become stale as Flutter and strong mode converge.  We'll be better off if people don't have these settings baked into their projects when that happens.